cypress/e2e/course/studentCreateEditAndDeleteGroupInCourse.feature

```diff
@@ -0,0 +1,74 @@
+@regression_test
+@stable_test
+@schedule_run
+@group-D
+@pr
+Feature: Course - Teacher can create, edit and delete a group in the course
+
+    Scenario Outline: user creation, course creation, and creating,editing and deleting a group
+
+        # pre-condition: creating all users and creating course
```

Comment:
this pre condition is only for creating all users, so the texts 'creating course' in this line can be removed?